### PR TITLE
User can view bracket with their team selections - Adrian Arias

### PR DESCRIPTION
**Pull request by adrianm-arias**
issue-10

### CODE REVIEW

File: client/pages/bracket.jsx [hunk 1]
```diff
@@ -81,89 +116,351 @@ export default class Brackets extends React.Component {
...
+    }
+    if (bracketRound === 'final') {
+      for (const property in koStage) {
+        if (property === 'game63') {
+          if (koStage[property] === teamId) {
+            return true;
+          }
+        }
+      }
+    }
```
Comment (thebearingedge) [inline client/pages/bracket.jsx]: I believe we can simplify the logic here, no loops required.

```js
if (bracketRound === 'roundof16') {
  return koStage[property] === teamId;
}
```

File: client/pages/bracket.jsx [hunk 2]
```diff
@@ -81,89 +116,351 @@ export default class Brackets extends React.Component {
...
+    if (event.target.checked) {
+      for (const property in koStageCopy) {
+        if (property === event.target.name) {
+          koStageCopy[property] = teamId;
+        }
+      }
+      this.setState({
+        knockoutStage: koStageCopy
+      });
+    }
```
Comment (thebearingedge) [inline client/pages/bracket.jsx]: I believe we can do this part without the loop too.

```js
koStageCopy[event.target.name] = teamId;
```

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: client/pages/bracket.jsx (hunk 1 above)
Comment (adrianm-arias) [reply client/pages/bracket.jsx]: Hey, I tried the above solution, it does not work because 'koStage[property]' is being defined from the for in loop.

Comment (thebearingedge) [reply client/pages/bracket.jsx]: oh oh, i'm sorry my example is incorrect. I meant to use the property key directly

```js
if (bracketRound === 'roundof16') {
  return koStage.game63 === teamId
}
```

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**